Cargo.toml

```diff
@@ -51,6 +51,7 @@ bevy_winit = ["bevy_internal/bevy_winit"]
 trace_chrome = ["bevy_internal/trace_chrome"]
 trace = ["bevy_internal/trace"]
 wgpu_trace = ["bevy_internal/wgpu_trace"]
+debug_draw = ["bevy_internal/debug_draw"]
```

Comment:
```suggestion
debug_draw = ["bevy_internal/bevy_debug_draw"]
```
I think it must be named this, to succesfully compile.

Reply:
This does not seem to help. The other crates do not require a `bevy_` prefix either.

Reply:
It is neccesary here, as you added it to `bevy_internal` this way:
```
bevy_debug_draw = { path = "../bevy_debug_draw", optional = true, version = "0.4.0" }
```
See the `bevy_` prefix?
Those that don't need it, also don't have the prefix:
```
wgpu_trace = ["bevy_wgpu/trace"]
trace = [ "bevy_app/trace", "bevy_ecs/trace" ]
trace_chrome = [ "bevy_log/tracing-chrome" ]
```

For me applying that change allows cargo to compile, though there are still several compile errors:
* The `With<>` I suggested below still needs to be imported.
* The trait functions in the gizmo file, do not need a `pub`

But at least it's progress.

Reply:
After fixing up the example, it also compiles successfully.
(Though I don't know why no Lines are drawn)
![debug_lines](https://user-images.githubusercontent.com/66798382/110967748-05f0e980-8357-11eb-8edd-c551e3b2aa66.PNG)

Reply:
I have sent you a PR on your Fork, with all the fixes.

Reply:
Thank you @MinerSebas .
I merged the PR on my fork and have the same behavior.
Will try and find the bug.